lib/symbol-store.coffee

```diff
@@ -97,10 +97,11 @@ class Symbol
 
     unless @type?
       typePriority = 0
-      for type, options of config
+      for type in Object.keys(config)
+        options = config[type]
         continue unless options.selectors?
         @metadataByPath.forEach ({scopeChains}) =>
-          for scopeChain, __ of scopeChains
+          for scopeChain in Object.keys(scopeChains)
```

Comment:
It does, but the alternative is a deoptimized method due to CoffeeScript hoisting the loop variables.  Pick your poison.